test/end2end/stream-2.0.1.json

```diff
@@ -1075,231 +1077,6 @@
           "col": 9,
           "description": "Unnecessary new keyword."
         },
-        {
```

Comment:
Why did all of these disappear?

Reply:
either a change in the analyzer or a change in the dependencies